src/Repository/ProfileField.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+namespace Friendica\Repository;
+
+use Friendica\BaseModel;
+use Friendica\BaseRepository;
+use Friendica\Collection;
+use Friendica\Database\Database;
+use Friendica\Database\DBA;
+use Friendica\DI;
```

Comment:
Inside dynamic classes, we should avoid calling `DI::`, because we're already inside the DI-logic. Dice will handle everything if we add the dependencies to the constructor :-)

Reply:
There were two separate things that made this happen:
- When a dependency is used in a single method, I'm tempted to pass the object directly in the method parameters
- The migration method used to be in `update.php` before being moved back to the repository  because it's used in user account import.

But yeah, I get it.